pkg/cmd/create/create.go

```diff
@@ -45,6 +48,10 @@ import (
 	kserve "knative.dev/serving/pkg/client/clientset/versioned"
 )
 
+const (
+	timedOut = "UPGRADE FAILED: timed out waiting for the condition'"
```

Comment:
Is the second character from the end "'" intended? Looks a bit like a typo.

Reply:
Good point